README.md

```diff
@@ -110,6 +112,22 @@ And then run it (this example assumes you'll want to use the default port):
 $ docker run -p 50051:50051 xain-fl
 ```
 
+### Docker-compose
+
+Development
+---
+
+```shell
+$ docker-compose -f docker-compose-dev.yml up
+```
+
+Release
+---
+
+```shell
+$ docker-compose up
```

Comment:
The release command does not work on my machine.
I get:
```
xain-fl_1  | xain_fl-0.2.0
xain-fl_xain-fl_1 exited with code 0
```

Reply:
looks like you have another image with the same name (probably from building directly from the Dockerfile?). Could you try again after removing the containers called `xain-fl`, `xain-fl_xain-fl_1` ?

Reply:
Ok it works after I have removed the images.

Reply:
But now I get an error when run `docker-compose -f docker-compose-dev.yml up`

can we rename the `xain-fl` service in docker-compose-dev.yml to `xain-fl-dev`?

Reply:
Done! 
Renamed minio and the networks with the same prefix, to have a clearer separation between the two